src/GitHub.VisualStudio/UI/Views/Controls/RepositoryCloneControl.xaml.cs

```diff
@@ -60,24 +63,60 @@ public RepositoryCloneControl()
             };
         }
 
-        static ListCollectionView CreateRepositoryListCollectionView(IEnumerable<IRepositoryModel> repositories)
+        ListCollectionView CreateRepositoryListCollectionView(IEnumerable<IRepositoryModel> repositories)
```

Comment:
It was static before but with these changes, but now `RepositoryGroupDescription` needs a reference to the `RepositoryCloneControl` (i.e. `this`) in order to collapse the existing group when a second group is added.